src/base-command.ts

```diff
@@ -92,11 +99,88 @@ export default abstract class BaseCommand extends Command {
     return super.parse(options, [...args, ...flags]);
   }
 
+  async _filterSensitiveSentryMetadata(sentry_metadata: any): Promise<any> {
+    const context = this.constructor as any;
+    return Object.entries(sentry_metadata).map(([key, value]) => {
+      if (context.non_sensitive.has(key)) {
+        return { [key]: value };
+      } else if (sentry_metadata.sensitive.has(key)) {
```

Comment:
We do not need both lists. Either something is sensitive or its not. We should start with the assumption everything is sensitive. Then if its in the non-sensitive list we are willing to report on it.

Reply:
Removed sensitive/non_sensitive sets, and added flag/arg property 'non_sensitive': boolean instead, which can be used by the BaseCommand class to filter out flags/args that have do not have the property 'non_sensitive'.